src/osdc/Striper.h

```diff
@@ -89,7 +89,7 @@
 	CephContext *cct, ceph::buffer::list& bl,
 	const std::vector<std::pair<uint64_t,uint64_t> >& buffer_extents);
       void add_partial_result(
-	  CephContext *cct, ceph::buffer::list&& bl,
+	  CephContext *cct, ceph::buffer::list& bl,
```

Comment:
Nit: why can't this keep the move semantics?  Can the `LightweightBufferExtents` version of `add_partial_sparse_result` switch to use move semantics as well if that was the rationale?

Reply:
there are 4 versions of this function, this is the only one using rvalue
anyhow, a version with an lvalue is trying to call this function and this is the problem.
cannot convert lvalue reference to rvalue reference

Reply:
Yep, I think this works. changes pushed.